drivers/mfd/Kconfig.rp1

```diff
@@ -0,0 +1,8 @@
+# Copyright (c) 2024 Junho Lee <junho@tsnlab.com>
+# SPDX -License-Identifier: Apache-2.0
+
+config MFD_RP1
+	bool "RP1 peripheral controller driver"
+	default y
```

Comment:
```suggestion
config MFD_RP1
	bool "RP1 peripheral controller driver"
	default y
	depends on DT_HAS_RASPBERRYPI_RP1_ENABLED
```

Reply:
I applied this change. Thank you for your suggestion.